istioctl/cmd/remove-from-mesh_test.go

```diff
@@ -139,6 +139,20 @@ func TestRemoveFromMesh(t *testing.T) {
 			expectedException: true,
 			expectedOutput:    "Error: expecting deployment name\n",
 		},
+		{
+			description:       "Invalid namespace - system namespace not allowed",
+			args:              strings.Split("experimental remove-from-mesh deployment local-path-storage", " "),
+			expectedException: true,
+			namespace:         "local-path-storage",
+			expectedOutput:    "Error: namespace local-path-storage is a system namesapce and has no Istio sidecar injected\n",
+		},
+		{
+			description:       "Invalid namespace - system namespace not allowed",
+			args:              strings.Split("experimental remove-from-mesh service istio-ingressgateway", " "),
+			expectedException: true,
+			namespace:         "istio-system",
+			expectedOutput:    "Error: namespace istio-system is a system namesapce and has no Istio sidecar injected\n",
```

Comment:
Maybe a better message if the user tries to remove sidecar from the istio-system namespace?